libafl_bolts/src/shmem.rs

```diff
@@ -680,24 +680,16 @@ pub mod unix_shmem {
                         0o600,
                     );
                     if shm_fd == -1 {
-                        println!(
-                            "errno : {:?}",
-                            std::io::Error::last_os_error().raw_os_error().unwrap()
-                        );
                         return Err(Error::unknown(format!(
-                            "Failed to shm_open map with id {filename_path:?}",
+                            "Failed to shm_open map with id {filename_path:?}, Last OS error :{:?}",std::io::Error::last_os_error(),
                         )));
                     }
 
                     /* configure the size of the shared memory segment */
                     if ftruncate(shm_fd, map_size.try_into()?) != 0 {
-                        println!(
-                            "errno : {:?}",
-                            std::io::Error::last_os_error().raw_os_error().unwrap()
-                        );
                         shm_unlink(filename_path.as_ptr() as *const _);
                         return Err(Error::unknown(format!(
-                            "setup_shm(): ftruncate() failed for map with id {filename_path:?}",
+                            "setup_shm(): ftruncate() failed for map with id {filename_path:?},Last OS error :{:?}",std::io::Error::last_os_error(),
```

Comment:
Space here, also lowercase `L`?

Reply:
should I add space after each `,`  and make all the `L` 's lowercase?